packages/cli/src/utils/deploy.ts

```diff
@@ -496,8 +512,15 @@ export async function deploy(
     try {
       const gasLimit = await contract.estimateGas[func].apply(null, args);
       console.log(chalk.gray(`executing transaction: ${functionName} with nonce ${nonce}`));
+      const _baseTxConfig = { gasLimit, nonce: nonce++ };
+      const txConfig = Object.assign(
+        _baseTxConfig,
+        maxPriorityFeePerGas !== undefined && maxFeePerGas !== undefined
+          ? { maxPriorityFeePerGas, maxFeePerGas }
+          : { gasPrice }
+      );
```

Comment:
same here, since we only set either (maxFeePerGas and maxPriorityFeePerGas) or (gasPrice), with the other one staying undefined, we don't need this conditional and can just pass all three variables to the function